Wikipedia/Code/SectionEditorWebViewMessagingController.swift

```diff
@@ -318,7 +307,7 @@ extension SectionEditorWebViewMessagingController {
                     return 11
                 case .redo:
                     return 12
-                case .debug:
+                case .progress:
```

Comment:
i stuck `progress` here so we wouldn't have a gap (the number 13 wasn't wired up to any button tags so nicer to just re-use it for `progress`)